languages/javascript/exercises/concept/array-destructuring/.docs/after.md

```diff
@@ -0,0 +1,82 @@
+Array destructuring assignment is a concise way of extracting values from an array. Its syntax is similar to an array literal expression, but on the left-hand side of the assignment instead of the right.
+
+```javascript
+const frenchNumbers = ['quatre-vingts', 'quatre-vingt-dix', 'cent']
+const [80, 90, 100] = frenchNumbers
```

Comment:
This actually doesn't compile! 

```suggestion
const [french80, french90, french100] = frenchNumbers
```

Reply:
Yikes, I really should've caught this one. Changed.